### PR TITLE
Create ayy.sh

### PR DESCRIPTION
**Pull request by ThatGamerBlue**

### CODE REVIEW

File: ayy.sh [hunk 1]
```diff
@@ -0,0 +1 @@
+echo ayy lmao
```
Comment (DjayGaming) [inline ayy.sh]: lol

Comment (Pixar87) [inline ayy.sh]: *c;lick*
Nice!

**Comment by DjayGaming** [comment]
Yea hehe

File: ayy.sh (hunk 1 above)
Comment (DjayGaming) [inline ayy.sh]: who wrote this +echo thing was it you or me that wrote that

Comment (Pixar87) [inline ayy.sh]: The + sign before a line of text is added by GitHub to indicate this is a line added in a pull request.

**Pull request merged**